### DISCUSSION TITLE
oh-my-zsh

### DISCUSSION THREAD

**Discussion by c0destep** [catppuccin/catppuccin#2007] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Poderia haver os temas para quem usa oh-my-zsh. Exemplo: Mocha

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
there are already themes for [starship](https://github.com/catppuccin/starship) and [zsh-syntax-highlighting](https://github.com/catppuccin/zsh-syntax-highlighting)
also, i dont believe catppuccin would have its own like prompt
with starship and such, they give you the color palette to use with your own prompt, instead of just giving you the config for the example prompt

**Reply by c0destep** [reply]
Thanks for your help
